serve-api/handlers/health.go

```diff
@@ -0,0 +1,25 @@
+package handlers
+
+import (
+	"fmt"
+
+	"gorm.io/gorm"
+)
+
+func DbStatusCheck(db *gorm.DB) (bool, string) {
+	database, err := db.DB()
+	if err != nil {
+		return false, "Internal Error"
+	}
+
+	if err := database.Ping(); err != nil {
+		return false, fmt.Sprintf("Database Connection Error: %s", err)
```

Comment:
hey @xaithe - not sure if the project is using strings intentianlly, so excuse me if im wrong! you can do `fmt.Errorf` which will give you an `error`, or in go 1.20+ there is `errors.Join` to merge together more than one

Reply:
The intention here is to have a message to send back to the main app when it hits the health check, so I used a string. The go api isn't actually active though so was more doing this for golang practice 🙂